ddtrace/tracer.py

```diff
@@ -283,10 +283,15 @@ def get_log_correlation_context(self):
         if self.enabled:
             span = self.current_span()
 
+        if span and span.service:
+            service = span.service
+        else:
+            service = config.service
+
         return {
             "trace_id": str(span.trace_id) if span else "0",
             "span_id": str(span.span_id) if span else "0",
-            "service": config.service or "",
+            "service": service,
```

Comment:
We want to retain an empty string as the default as we do for `version` and `env`:

```suggestion
            "service": service or "",
```